package.json

```diff
@@ -10,9 +10,11 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.2",
     "@fortawesome/free-solid-svg-icons": "^5.2.0",
     "@fortawesome/react-fontawesome": "^0.1.0",
-    "@gen3/ui-component": "0.2.0",
+    "@gen3/ui-component": "0.2.2",
     "@storybook/addon-actions": "^5.0.0",
     "@storybook/react": "^5.0.0",
+    "babel-core": "^6.26.0",
```

Comment:
these things shouldn't be necessary? they are old versions of babel

Reply:
✓